### DISCUSSION TITLE
line 222, in resize_seam_masks     for idx, seam_mask in enumerate(seam_masks): TypeError: 'NoneType' object is not iterable

### DISCUSSION THREAD

**Discussion by davidhardeniv-code** [OpenStitching/stitching#233] [Q&A]
I was wondering if anyone has got this error and if you did how did you fix it. I have been working on it for a bit and when I tried my code that works with it in another project the same script doesn't work even though no changes were made. Any help would be great!!

**Comment by lukasalexanderweber** [comment]
You have to find out why seam_masks is None.. this might be a bug, but we definitely need more information to hunt it
